examples/metrics-http/src/main/java/io/opentelemetry/example/httpserver/application/Application.java

```diff
@@ -0,0 +1,70 @@
+package io.opentelemetry.example.httpserver.application;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import io.oopentelemetry.example.httpserver.instrumented.InstrumentedHttpServer;
+import io.oopentelemetry.example.httpserver.instrumented.ProcessMetrics;
+import io.oopentelemetry.example.httpserver.instrumented.ServerRoomMetrics;
+import io.opentelemetry.exporter.logging.LoggingMetricExporter;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Collections;
+
+public class Application {
+  public static void main(String[] args) throws IOException {
+    // First we grab access to all global metrics.
+    final SdkMeterProvider globalMetrics =
+        SdkMeterProvider.builder()
+            // Make sure we hijack "global" metrics to ensure our HTTP library metrics are reported.
+            .buildAndRegisterGlobal();
+
+    // Next we instantiate the ProcessMetrics against this global metrics.
+    final ProcessMetrics processMetrics = ProcessMetrics.create(globalMetrics);
+    // Next we attach an interval for pull-metrics that wires our global metrics to an exporter.
+    IntervalMetricReader globalReader =
+        IntervalMetricReader.builder()
+            // Collect metrics every 5 seconds.
+            .setExportIntervalMillis(5000)
+            // Capture ALL global metrics from any library.
+            .setMetricProducers(Collections.singleton(globalMetrics))
+            // For now just log the metrics.
+            .setMetricExporter(new LoggingMetricExporter())
+            .build();
+
+    // Next we want to create the server room metrics at a LONGER poll interval.  In
+    // today's API this requires a separate metric provider + interval reader.
```

Comment:
There is similar limitation in OpenTelemetry Python today.

Reply:
For the stake of proving the API, let's assume the SDK will allow you to configure "something" per Instrument (matching name, and instrumentation library name) that can do that.

Point taken that this is needed, but is an SDK config in my opinion.